xcube/core/varexpr/context.py

```diff
@@ -171,6 +154,21 @@ def evaluate(self, var_expr: str) -> xr.DataArray:
         return result
 
 
-class VarExprValidator(ast.NodeTransformer):
-    def visit_Lambda(self, node):
-        raise VarExprError("lambda expressions are not supported")
+def _format_callable(name: str, fn: Union[np.ufunc, Callable]):
```

Comment:
What is the reason for putting this outside of the class?

Reply:
It is was a private static method and now it is a private function, calling is shorter as it does not require context object.